sbtix-tool.nix

```diff
@@ -37,6 +37,7 @@ let
 
     #the global plugins directory must be writeable
     SBTIX_GLBASE_DIR="$HOME/.sbtix"
+    echo "Using SBTIX_GLBASE_DIR $SBTIX_GLBASE_DIR"
```

Comment:
```suggestion
```

It's a predictable location, and not particularly useful for users.

Reply:
indeed. I think I've added it for troubleshooting.